content/realtime.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: Realtime data collection
+---
+
+import CreateWebhookSnippet from '/snippets/code-examples/create-webhook.mdx';
+
+PathStack supports realtime data access through many of our providers, you can find the list of supported providers
+[here](/content/providers). We provide a direct connection to the telematics provider so you can get live data as soon
+as possible.
+
+## Applications
+
+Applications for realtime data include:
+
+- **Realtime Tracking of Vehicles in Your Fleet**
+
+  We can provide the live location, as well as attributes such as speed
+  and heading if they are available.
+
+<Note>Trip milestones is an upcoming feature. Please contact us on 1300 822 370 to find out more!</Note>
```

Comment:
I'm not a fan of including vapourware in the API docs.  I don't think milestones belong here yet.